e2e/components/CodeSnippet/CodeSnippet-test.avt.e2e.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('CodeSnippet', () => {
+  test('accessibility-checker CodeSnippet inline @avt', async ({ page }) => {
```

Comment:
Minor thing, but you can put the `@avt` flag in the describe block and then you don't have to add it to every test. As this file grows could be helpful to ensure everything in the block will get picked up by the test runner.

```suggestion
test.describe('CodeSnippet @avt', () => {
  test('accessibility-checker CodeSnippet inline', async ({ page }) => {
```

Reply:
Makes sense!